x-pack/plugins/reporting/server/browsers/chromium/driver_factory/constants.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const BROWSER_LAUNCH_TIME_TO_WAIT = 5 * 1000;
+
+export const getBrowserLaunchTime = () => BROWSER_LAUNCH_TIME_TO_WAIT;
```

Comment:
I don't think the chosen approach is correct. It is not a good idea to have a helper function returning a constant just for unit tests.
Have you tried [`jest.useFakeTimers`](https://jestjs.io/docs/jest-object#jestusefaketimersimplementation-modern--legacy)?
If that doesn't work, then the second option will be using [RxJS marbles](https://rxjs.dev/guide/testing/marble-testing#time-progression-syntax).

Reply:
OK, I'll look into that 👍🏻